lib/create-testing-library-rule/detect-testing-library-utils.ts

```diff
@@ -326,6 +333,35 @@ export function detectTestingLibraryUtils<
 			return getImportModuleName(importedCustomModuleNode);
 		};
 
+		const getUserEventImportIdentifier = (node: ImportModuleNode | null) => {
+			if (!node) {
+				return null;
+			}
+
+			if (isImportDeclaration(node)) {
```

Comment:
This function should always receive a `node`. If the node is not available, then avoid calling this function in the corresponding code.
```suggestion
		const getUserEventImportIdentifier = (node: ImportModuleNode) => {
			if (isImportDeclaration(node)) {
```